Snowflake.Data.Tests/Snowflake.Data.Tests.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp2.0;net46</TargetFrameworks>
+    <TargetFrameworks>netcoreapp2.2;net46</TargetFrameworks>

```

Comment:
For the 2nd CVE in the list, upgrade project to .NETCore App 2.2 using .NETCore SDK 2.2.110